### ISSUE TITLE
can't build with hexlog

### ISSUE THREAD

**Issue by sonygod** [darmie/cabasa#1]
Error: ./src/hex/log/LoggerConfig.cpp: In member function ‘virtual hx::Val hex::log::LoggerConfig_obj::__Field(const String&, hx::PropertyAccess)’:

**Comment by darmie** [comment]
@sonygod C++ target is broken for hexlog version I installed.  

I will take a look.

_Btw, I just pushed another commit for [Wasp](https://github.com/darmie/wasp)_

**Comment by darmie** [comment]
@sonygod I have fixed the hexlog issue

Install the alpha version

`haxelib install hexlog 1.0.0-alpha.7`

**Comment by sonygod** [comment]
still got nothing trace or file generate?

have you test your project and then push into github?

**Comment by sonygod** [comment]
![image](https://user-images.githubusercontent.com/748320/73420893-0ac72980-435f-11ea-9730-18fcfd8cdc55.png)

**Comment by darmie** [comment]
I have not tested Cabasa with cpp, I have only tested Wasp with cpp.

**Comment by darmie** [comment]
@sonygod use `build.hxml`

In `test.hxml` I was trying to use a new C++ and Hashlink binder which is not working. 

Apologies for the confusion. 

Also please note that C++ version of Cabasa doesn't work yet.

**Comment by sonygod** [comment]
oh....

which target have you test before?

**Comment by darmie** [comment]
C# and Java

**Comment by darmie** [comment]
@sonygod I just made a new PR merge. You can pull from master and run it again for C++ target.

C++ target now working

**Comment by sonygod** [comment]
I'll take a look later..

**Comment by sonygod** [comment]
![$RY A)4LC7T~%FUQ6X@HC8E](https://user-images.githubusercontent.com/748320/73446016-f56df180-4396-11ea-9b3f-813651ad31a9.jpg)

there are still something error ,maybe platform  issue?

ubuntu  18.04

@darmie

**Comment by darmie** [comment]
@sonygod please update your version of Wasp

**Comment by sonygod** [comment]
after update.it's seem only 

 call_fib(raw5);

not work.

erveryhing  seem ok 


![Y05U`ZRA}L`4EK%18)V2~6X](https://user-images.githubusercontent.com/748320/73454145-71246a00-43a8-11ea-83b3-cd2014ab93fc.png)

**Comment by darmie** [comment]
Yes I am aware. I commented it out because  I am still investigating it.

**Comment by sonygod** [comment]
are you going to support wasm build by hxcpp?

the current version I did test seems not support hxcpp's wasm .

[wasm.zip](https://github.com/darmie/cabasa/files/4138386/wasm.zip)

**Comment by darmie** [comment]
@sonygod Cabasa is designed to support the MVP version of wasm, it depends on what tool you used to compile hxcpp code to wasm.  I suggest you use Binaryen.

**Comment by sonygod** [comment]
I jsut use hxcpp defalut toolchain.xml with emcc ,

is this not Binaryen?

**Comment by darmie** [comment]
I don't know. I will focus on hxcpp specifics when I have much time. Right now, I use wasm builds from [WebAssembly Studio](https://webassembly.studio/) for my tests.
